### DISCUSSION TITLE
How to get "brain" badge on my profile

### DISCUSSION THREAD

**Discussion by levintech** [community/community#30117] [Other Feature Feedback, Questions, & Ideas] [answered]
Dear, everyone, 
I wanna know how to get "Brain" badge on my github profile. 
If you know how to do it, please guide me,
Thanks

**Comment by solyasa** [comment, marked as answer]
- Answer at least 2 discussions
- Take note, you can't mark your own answers to your own questions. They have to be marked by another user
- Discussions should be in a public repository
- There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead

 Hope this answer has helped you, and all the best in getting the achievement
